front/app/api/navbar/useAddNavbarItem.ts

```diff
@@ -0,0 +1,41 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import navbarKeys from './keys';
+import { INavbarItemResponse, INavbarItemAdd } from './types';
+import { IItemNotInNavbar } from 'utils/navbar';
+import customPagesKeys from 'api/custom_pages/keys';
+
+const addNavbarItem = async (item: IItemNotInNavbar) => {
```

Comment:
```suggestion
const addNavbarItem = (item: IItemNotInNavbar) => {
```

Reply:
I see async functions in other useAdd... hooks, but we're not awaiting anything so not needed, right?

Reply:
Correct, nice catch!